hotkeys.js

```diff
@@ -25,6 +26,10 @@ _.extend(Hotkeys.prototype, {
             check(combo, String);
         }
 
+        if(!_.isString(reference)){
```

Comment:
i would also default to undefined or maybe just leave out the check as mousetrap seems to handle this, docs state the following:

> It is recommended that you leave this argument out if you are unsure. Mousetrap will look at the keys you are binding and determine whether it should default to keypress or keydown.